lib/llvm/Makefile

```diff
@@ -97,13 +97,8 @@ LLVM_INSTALL_DIR := $(ROOT_DIR)/dist
 LLVM_BUILD_ENGINE := "Unix Makefiles"
 LLVM_BUILD_TYPE := Release
 
-LLVM_LINKER := bfd
-#ifeq (llvm-3.9.1,$(LLVM_TARGET))
-#  # 3.9.1 doesn't supprot -DLLVM_USE_LINKER so make it empty to supress a warning
-#  LLVM_USE_LINKER :=
-#else
-#  LLVM_USE_LINKER := -DLLVM_USE_LINKER=$(LLVM_LINKER)
-#endif
+#LLVM_LINKER := bfd
```

Comment:
Let's add a little note why you might want `bfd`, or remove the comment?

Reply:
Good idea